guides/V1_message_main.md

```diff
@@ -22,3 +22,4 @@ Watson Work Services represents a Message with this object.
 | content   | String | The body of the message |
 | contentType | String | Mime type of the message content  |
 | annotations | [String] | A set of optional objects/attachments that are added to a message, called annotations. Each annotation represents meta data that are related to the message. An annotation is represented by a structured format, which can be viewed as a template. There are different annotation types, some are used to store results of cognitive analysis to a message for example. An annotation can be added at message create time or later, as an update.  Apps are currently limited to create only one type of annotation called `generic` |
+| reactions | [Reaction] | A set of optional objects that are added to a message, called reactions. Each reaction represents how users reacted (by emoji) to the message. A Reaction is represented by the reaction (String emoji), the count (Int number of users that reacted with the same reaction), and viewerHasReacted (Boolean true the calling user reacted with given reaction, or false didn not). This requires the `EXPERIMENTAL` value in `x-graphql-view header`. |
```

Comment:
You can ignore my comment regarding the description for the mutations (adding count and user has reacted) since its explained here although I don't think it hurts having it in both places.

Also there seems to be a typo:
> (Boolean true the calling user reacted with given reaction, or false didn not).